development/README.md

```diff
@@ -282,4 +282,191 @@ $BUILD_SCRIPTS/qgis-dev-install.sh $BUILD_DIR
 
 ## Configure/build/install QGIS in Qt Creator.app
 
-TODO
+### Install QtCreator 4.0 or better
+
+* Grab the installer from https://www.qt.io/download-open-source/
+* Run the installer. You ONLY need the qt-creator package so disable the Qt5 and don’t install Qt libs (we will use the qt4 installed by brew)
+* Once the installer is run open qt-creator
+* Make a directory in your QGIS source tree called ``build``
+* Open the top level ``CMakeLists.txt`` in QtCreator
+
+### Making a kit
+
+You need to modify the desktop kit to use your brew install qt5:
+
+``Preferences -> build and run -> Qt Versions -> Manual -> Add``
+
+Set the executable to :
+
+``/usr/local/bin/qmake``
+
+``Preferences -> build and run -> Cmake -> Manual -> Add``
+
+* **Name:** Brew CMake
+* **Path:** /usr/local/bin/cmake
+
+
+``Preferences -> build and run -> Kits -> Manual Desktop``
```

Comment:
Recommend to always rewrite Homebrew paths to their 'opt_prefix' alternative, so as to avoid problems with your kit due to incremental Homebrew upgrades, e.g. `Cellar/ccache/3.3.3` -> `Cellar/ccache/3.3.3_1`. Such upgrades change the install path, regardless of whether there is actually any change to the formula's actual package version.

In this instance, the 'opt_prefix' change is:

```
Compiler path: /usr/local/opt/ccache/libexec/clang
Compiler path: /usr/local/opt/ccache/libexec/clang++
```

The 'opt_prefix' will always be a symlink to the most current installed keg, e.g. `/usr/local/Cellar/ccache/<whatever-version>`